modules/tensor_mechanics/test/tests/beam/dynamic/tests

```diff
@@ -1,173 +1,287 @@
 [Tests]
+  design = "C0TimoshenkoBeam.md"
+  issues = "#10313"
   [./dyn_euler]
     type = 'Exodiff'
     input = 'dyn_euler_small.i'
     exodiff = 'dyn_euler_small_out.e'
+
+    requirement = "The dynamic system shall correctly predict the natural frequencies"
```

Comment:
The mechanics system shall correctly predict the natural frequencies of an Euler-Bernoulli beam modeled using beam elements with consistent mass/inertia.

Reply:
I think I would change all of these to refer to the 'mechanics' system, rather than the dynamic system.